dh.cpp

```diff
@@ -148,11 +148,15 @@ openssl_unique_ptr<DH> make_dh (const unsigned char* prime, size_t prime_len, co
 		throw Openssl_error(ERR_get_error());
 	}
 
-	if ((dh->p = BN_bin2bn(prime, prime_len, NULL)) == NULL) {
+  BIGNUM* p = NULL;
+  BIGNUM* g = NULL;
+  if ((p = BN_bin2bn(prime, prime_len, NULL)) == NULL) {
```

Comment:
Also here.

Reply:
I think I got all of the space instead of tab indentation issues.